version_test.go

```diff
@@ -5,6 +5,10 @@ import (
 	"encoding/json"
 	"fmt"
 	"testing"
+
+	yamlGoccy "github.com/goccy/go-yaml"
```

Comment:
it is not needed to test that feature.

Reply:
I wanted to test with different libraries that use different logic to marshal and unmarshal.

For example, github.com/goccy/go-yaml has this interface:

```
type BytesMarshaler interface {
	MarshalYAML() ([]byte, error)
}
```

gopkg.in/yaml.v2 and gopkg.in/yaml.v3 have this interface:

```
type Marshaler interface {
	MarshalYAML() (interface{}, error)
}
```

So in order to make yaml marshalling and unmarshalling work, we can't just implement a MarshalYAML method because the interfaces conflict.